container-images/mds-agency/tsconfig.build.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "references": [
+    { "path": "../../aws-lambda/aws-serverless-express-handler/tsconfig.build.json" },
```

Comment:
Again, I don't think we need to reference `aws-lambda/aws-serverless-express-handler` at all here.